### PR TITLE
rebase on docusaurus

### PR DESCRIPTION
**Pull request by usrbinkat**
Preview screenshots:
image
image
image
image

### CODE REVIEW

File: blog/authors.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+afflom:
+  name: Alex Flom
+  title: Docusaurus maintainer
```
Comment (afflom) [inline blog/authors.yml:3]: Systems Architect

File: blog/authors.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  url: https://github.com/afflom
+  image_url: https://github.com/afflom.png
+
+jpower432:
+  name: Jennifer Power
+  title: Docusaurus maintainer
```
Comment (afflom) [inline blog/authors.yml:9]: Lead Developer

File: blog/authors.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  url: https://github.com/jpower432
+  image_url: https://github.com/jpower432.png
+
+sabre1041:
+  name: Andrew Block
+  title: Docusaurus maintainer
```
Comment (afflom) [inline blog/authors.yml:15]: Project Manager

File: blog/2022-08-21-welcome/index.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+slug: welcome
+title: Welcome
+authors: [afflom, jpower432, sabre1041, usrbinkat]
+tags: [facebook, hello, docusaurus]
```
Comment (afflom) [inline blog/2022-08-21-welcome/index.md:5]: Remove

File: docs/intro.md [hunk 5]
```diff
@@ -1,20 +1,12 @@
...
-:toclevels: 3
-:sectnumlevels: 3
+# FAQ
 
+Let's cover some basics of UOR.
 
-### What is UOR? 
+## What is UOR?
 
 Universal Object Reference (UOR) provides a single format to publish, retrieve, and interact with any content. By “any content” we mean literally anything.
```
Comment (afflom) [inline docs/intro.md:11]: `UOR is a framework used to generalize multiple streams of content management into a single content management mechanism and strategy. `

File: docs/intro.md [hunk 6]
```diff
@@ -25,7 +17,7 @@ Didn’t we say anything? A few examples of how UOR can be used are:
 - File Management  
 - Hosting static or dynamic web content  
 - Labeling and Dataset management  
-- AI model management - https://asciinema.org/a/wWCR7VY0zC97m3ibd3mKXcsJL  
+- [AI model management](https://asciinema.org/a/wWCR7VY0zC97m3ibd3mKXcsJL)
 - As a relational database  
 - As an object-relational database
```
Comment (afflom) [inline docs/intro.md:22]: Remove

File: docs/Quick Start/_category_.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Getting Started",
+  "position": 2,
+  "link": {
+    "type": "generated-index",
+    "description": "5 minutes to learn the most important Docusaurus concepts."
```
Comment (afflom) [inline docs/Quick Start/_category_.json:6]: Change to UOR

File: src/components/HomepageFeatures/index.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+
+const FeatureList = [
```
Comment (afflom) [inline src/components/HomepageFeatures/index.js:5]: Can these please be changed to the UOR pillars? https://docs.google.com/presentation/d/e/2PACX-1vSBbrsaCHP4irxT4s41YJREdEeAtvdgX8xrMwNpOh_hBZa6eKybUpD3FLr06f9QCKKSp9-Hgs47RIUC/pub?start=false&loop=false&delayms=10000&slide=id.g1445db85d7c_0_238

**Review by afflom — CHANGES_REQUESTED** [review]
This site is a major upgrade for us. Thank you for tackling this. The developer doc import is super slick. I tagged a few things that I thought might enhance this first version of the new site.

File: blog/authors.yml [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+  url: https://github.com/sabre1041
+  image_url: https://github.com/sabre1041.png
+
+usrbinkat:
+  name: Kat Morgan
+  title: Developer Advocate
```
Comment (afflom) [inline blog/authors.yml:21]: you can be whatever you want in this new world!

File: docs/intro.md [hunk 10]
```diff
@@ -25,7 +17,7 @@ Didn’t we say anything? A few examples of how UOR can be used are:
 - File Management  
 - Hosting static or dynamic web content  
 - Labeling and Dataset management  
-- AI model management - https://asciinema.org/a/wWCR7VY0zC97m3ibd3mKXcsJL  
+- [AI model management](https://asciinema.org/a/wWCR7VY0zC97m3ibd3mKXcsJL)
 - As a relational database
```
Comment (afflom) [inline docs/intro.md:21]: Remove references to relational and object relational databases

File: blog/authors.yml (hunk 9 above)
Comment (usrbinkat) [reply blog/authors.yml:21]: I am who I am

File: docs/intro.md (hunk 10 above)
Comment (usrbinkat) [reply docs/intro.md:21]: @afflom removing line 21, any other lines need removed?

Comment (afflom) [reply docs/intro.md:21]: 22 as well please

File: src/components/HomepageFeatures/index.js (hunk 8 above)
Comment (usrbinkat) [reply src/components/HomepageFeatures/index.js:5]: resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)

File: docs/Quick Start/_category_.json (hunk 7 above)
Comment (usrbinkat) [reply docs/Quick Start/_category_.json]: resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)

File: docs/intro.md (hunk 5 above)
Comment (usrbinkat) [reply docs/intro.md]: resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)

File: blog/2022-08-21-welcome/index.md (hunk 4 above)
Comment (usrbinkat) [reply blog/2022-08-21-welcome/index.md]: resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)

File: blog/authors.yml (hunk 3 above)
Comment (usrbinkat) [reply blog/authors.yml]: resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)

File: blog/authors.yml (hunk 2 above)
Comment (usrbinkat) [reply blog/authors.yml]: resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)

File: blog/authors.yml (hunk 1 above)
Comment (usrbinkat) [reply blog/authors.yml]: resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)

**Comment by afflom** [comment]
So much love here... ❤️ and LGTM

**Review by afflom — APPROVED** [review]

**Comment by sabre1041** [comment]
@usrbinkat attempted to build and run the container and hit the following error

```
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /src/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/src/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-08-25T21_16_41_885Z-debug-0.log
```

**Comment by usrbinkat** [comment]
@sabre1041 I was able to reproduce the same err using the podman commands. Root caused it as an issue of podman (on my mac) not mounting the local directory into the container at all. this is a new behavior in the last 24 hours, I will look into it.

Alternatively, `docker compose up` works still and uses the pre-built image from `ghcr.io/uor-framework/site`

**Review by sabre1041 — COMMENTED** [review]
LGTM

**Pull request merged**